testsuite/manualmode/src/test/java/org/jboss/as/test/manualmode/management/cli/ReloadOpsTestCase.java

```diff
@@ -108,55 +94,13 @@ public void testWriteAttribvuteWithReload() throws Exception {
         assertThat(value, is("false"));
     }
 
-    private void reloadServer(long timeout) throws Exception {
-        executeReload();
-        waitForLiveServerToReload(timeout);
```

Comment:
This waitForLiveServerToReload stuff shouldn't be needed. The executeReload method uses the high level CLI reload command, and that should already be waiting.